### PR TITLE
IMD and IMD reset sections (2.2 and 2.5)

### PR DESCRIPTION
**Pull request by nrivkin**

### CODE REVIEW

**Review by aehoppe — CHANGES_REQUESTED** [review]
We're actually using a TE Connectivity MICRO MATCH connector, I'd check the part number there.

**Comment by nrivkin** [comment]
Fixed connector type and added mention of fuse

File: esf_main.tex [hunk 1]
```diff
@@ -207,33 +207,74 @@ \subsubsection{Position in car}
...
 	\end{table}
+\end{center}
 
-\subsubsection{Wiring/cables/connectors/}
+
+
+
+
+\subsubsection*{2.2.2 Wiring/cables/connectors/}
+The  IMD uses the TYCO-MICRO MATE-N-LOK 1 x 2-1445088-8 connector and it's mate. It connects to the accumulator interface board, which sends the signal to a through a 28 AWG ribbon cable to the IMD latch using a 6-pin TE Connectivity MICRO MATCH 338068-6 connector.The 28 AWG wire is used because the current draw of the IMD is 150mA, which allows high gauge wire to be used. There is a 1 amp fuse protecting the high voltage wiring of the IMD. % I don't know the details of the fuse, but it is mentioned in both last year's ESF and in the ESF writing guide. It is visible in some of the plans for the electrical system. I need to know where I can find the details. Last year it was a 1 amp fuse, but it might be different this year. I am writing 1 amp for now, but it needs to be confirmed.
```
Comment (aehoppe) [inline esf_main.tex]: As per last year's ESF, please provide a \ref to a section you create in the appendix for the IMD datasheet with the important parameters (and a hyperlink to the full datasheet), as well as a \ref to a section in the appendix where the fuse information will go. I will fill in the fuse information later today.

File: esf_main.tex [hunk 2]
```diff
@@ -291,18 +332,37 @@ \subsubsection{Position in car/mechanical fastening/mechanical connection}
...
+
+
+% find out details of BMS
 
 \subsubsection{Wiring/cables/connectors}
 %Describe wiring, show schematics, describe connectors and cables used and show useful data regarding the wiring.  If not detailed in section 2.1, be sure to show how the device opens the shutdown circuit.
-TODO
+%TODO
+
+The IMD latch will be connected to the ?? board via a 4-pin Molex LLC Ultrafit 1722861104 with 20 AWG wire, and the ?? board using a  6-pin TE Connectivity MICRO MATCH 338068-6 connector with 28 AWG ribbon cable. The 20 AWG wire is used for the shutdown circuit and to supply power and ground to the IMD latch. The ribbon cable carries signals from the IMD.%On the ?? board it will connect , or another Molex product of the same series but with a different number of headers on each side of the connector.
```
Comment (aehoppe) [inline esf_main.tex]: What are these question marks for? Also we need to figure out fusing for those wires and current and temperature ratings. Please reference last year's ESF for the important information to put here.

**Review by aehoppe — CHANGES_REQUESTED** [review]

File: esf_main.tex (hunk 2 above)
Comment (nrivkin) [reply esf_main.tex]: Are we using the wiring found here? 
Mouser Part #:
571-19711119
Manufacturer Part #:
1-971111-9

http://www.mouser.com/ProductDetail/TE-Connectivity/1-971111-9/?qs=tNpDSWKagOPkPLjfSGNYAA%3d%3d

**Comment by nrivkin** [comment]
Added IMD datasheet, added refs, added what I could find about wire types. For 20 AWG (in the shutdown circuit) and 18 AWG (for power transmission) I do not know the exact wire we are using so I could not find voltage or temp ratings. I got current ratings from http://www.powerstream.com/Wire_Size.htm, and assuming our ribbon cable was the one found here, or at least had equivalent ratings.http://www.mouser.com/ProductDetail/TE-Connectivity/1-971111-9/?qs=tNpDSWKagOPkPLjfSGNYAA%3d%3d

File: esf_main.tex (hunk 1 above)
Comment (nrivkin) [reply esf_main.tex]: done

**Comment by aehoppe** [comment]
Please add that since the IMD will be in the accumulator, the wiring to it will be rated to at least 300V and at least 90C.

**Comment by aehoppe** [comment]
Closed because it was merged via a different PR

**Pull request closed without merging**